Makefile

```diff
@@ -12,11 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# We need all the Make variables exported as env vars.
+# Note that the ?= operator works regardless.
+.EXPORT_ALL_VARIABLES:
+
 # Enable Go modules.
-export GO111MODULE=on
+GO111MODULE=on
 
 REGISTRY ?= gcr.io/k8s-staging-gateway-api
-TAG ?= dev
+
+# These are overridden by cloudbuild.yaml when run by Prow.
+GIT_TAG ?= dev
```

Comment:
Do you have a reference to what these are overridden by?

Reply:
I've added some reference into the Makefile here.